linode/manager
# change: [M3-8024] - Update Linode disk action menu

## Description 📝
Updates the Linode disk action menu based on the latest UX mockups from Image Service Gen2.

## Changes  🔄
- Change copy from "Imagize" to "Create Disk Image"
- Clicking "Create Disk Image" now redirects to the Images Create page with Linode, Disk, and Label prefilled
- For Swap Images, the Create Image and Clone options should be disabled
- Updates unit tests accordingly

## Target release date 🗓️
7/8

## Preview 📷

| Before  | After   |
| ------- | ------- |
|  <video src="https://github.com/linode/manager/assets/122488130/0a973196-e055-4be3-849c-40463a0b5e90" /> | <video src="https://github.com/linode/manager/assets/122488130/d4ddb7ce-97d2-425f-bb34-15a65cabb047" /> |

## How to test 🧪

### Verification steps
- Navigate to the "Storage" tab under Linode details for one of your Linodes
- Click on the action menu for a disk and verify the new copy: "Create Disk Image"
- Verify clicking on "Create Disk Image" redirects to the Capture Image tab of Image Create with the Linode and Disk pre-filled
- Return to the action menu for a swap disk and verify the "Create Disk Image" option is disabled with the helper text "You cannot create image from Swap images." 

## As an Author I have considered 🤔

*Check all that apply*
